### DISCUSSION TITLE
Pre-PEFT time SDXL Lora fails and outputs black squares😭

### DISCUSSION THREAD

**Discussion by dibbla** [huggingface/diffusers#9201] [Q&A] [answered]
Hi guys,

I am using diffusers==0.24 for SDXL LoRA training. This is a resaecrh project and 0.24 is the highest version I can get to manipluate exposed LoRA layers.

However, as I setting everything to the defualt setting as in `/examples`, I keep getting all black squares. Is it a known bug in the past? How can I save my SDXL lora?

Many thanks in advance.

**Comment by asomoza** [comment]
Hi, you'll need to provide a minimal reproducible code with the error you're facing. For me it's not clear if you're doing inference or training, but if you're using a lora for inference we'll need the lora checkpoint too. 

Also if possible it's better if you also post some of the images with those black squares unless you mean that the image is completely black.

**Reply by dibbla** [reply, marked as answer]
> Hi, you'll need to provide a minimal reproducible code with the error you're facing. For me it's not clear if you're doing inference or training, but if you're using a lora for inference we'll need the lora checkpoint too.
> 
> Also if possible it's better if you also post some of the images with those black squares unless you mean that the image is completely black.

Hi @asomoza 

Thanks for the reply. Yes the images are completely black, and I am using the official implementations for both inference and training. My problem is (was), the validation process only outputs completely black images. What I did is redo all the dreambooth lora training again, with default code, default setting (and defualt dog dataset) and it works. 

I am not sure what is causing the problem but I will try to figure it out. Again thanks for the reply!
